Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/signature/core/DscDataParser.kt

```diff
@@ -1,12 +1,21 @@
 package de.rki.coronawarnapp.covidcertificate.signature.core
 
 import dagger.Reusable
+import de.rki.coronawarnapp.server.protocols.internal.dgc.DscListOuterClass
+import de.rki.coronawarnapp.util.toOkioByteString
+import org.joda.time.Instant
 import javax.inject.Inject
 
 @Reusable
 class DscDataParser @Inject constructor() {
 
-    fun parse(rawData: ByteArray): DscData {
-        throw NotImplementedError()
-    }
+    fun parse(rawData: ByteArray, updatedAt: Instant = Instant.EPOCH): DscData = DscData(
```

Comment:
can we  make default value `Instant.now()` ?

Reply:
we can, I don't have preferences